### PR TITLE
Missing translations and punctuation problems in problem-emails screen

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé

Fixes https://github.com/cds-snc/notification-planning/issues/1245

- Missing translation for `{{ _( ' using template ')}} '{{ notification["template"]["name"] }}'`
- Missing translation for `{{ _( ' in ')}} '{{ job.original_file_name }}'`
- Formatting `Less than 0.1%` with nbsp in FR

Also suggesting using more string templates to format the list of problems, so that translations are easier to track down in the future.

### CODE REVIEW

File: app/translations/csv/fr.csv [hunk 1]
```diff
@@ -1718,15 +1718,16 @@
...
-" in "," dans "
+"using template ‘{}’","via «{}»"
+"{count} problem email addresses","{count} problèmes"
+"1 problem email address","1 problème"
+"in ‘{}’","dans «{}»"
 "Only show problem addresses","N’afficher que les adresses problématiques"
 "Review Address","Réviser l’adresse"
 "Available until","Disponible jusqu’au"
 "Virus in attachment","Virus dans la pièce jointe"
 "Less than","Moins de"
```
Comment (whabanks) [inline app/translations/csv/fr.csv:1729]:
```suggestion
"Less than {}","Moins de {}"
```
FR translation wasn't getting picked up properly here.

File: app/templates/views/dashboard/review-email-list.html [hunk 2]
```diff
@@ -48,25 +48,20 @@ <h2 class="heading-medium mt-8">{{ _('Problem emails over the past 24 hours') }}
                 {% if job["bounce_count"] > 0 %}
                     <li>
                         <a href="{{ url_for('.view_job', service_id=current_service.id, job_id=job.id, status='permanent-failure', pe_filter='true') }}">
-                            {{ job["bounce_count"] }}
-                            {% if job["bounce_count"] > 1 %}
-                                {{_(' problem email addresses')}}
+                            {%- if job["bounce_count"] > 1 -%}
+                                {{ _('{count} problem email addresses').format(count=5) }}
```
Comment (whabanks) [inline app/templates/views/dashboard/review-email-list.html:52]:
```suggestion
                                {{ _('{count} problem email addresses').format(count=job["bounce_count"]) }}
```

**Review by whabanks — CHANGES_REQUESTED** [review]
Just a couple of suggestions and then this should be good to go!

File: app/translations/csv/fr.csv (hunk 1 above)
Comment (amazingphilippe) [reply app/translations/csv/fr.csv:1729]: Nice catch!

File: app/templates/views/dashboard/review-email-list.html (hunk 2 above)
Comment (amazingphilippe) [reply app/templates/views/dashboard/review-email-list.html:52]: This would've been catastrophic. Thanks for spotting!

**Review by whabanks — APPROVED** [review]
Looks good!

**Pull request merged**